src/routers/secure/items.py

```
@@ -221,17 +221,25 @@ async def add_items(request: Request, imdb_ids: str = None) -> MessageResponse:
+                return item.to_extended_dict(with_streams=False)
+            raise NoResultFound
+        except NoResultFound:
             if use_tmdb_id:
-                query = query.where(MediaItem.tmdb_id == id)
+                logger.debug(f"Item with TMDB ID {id} not found in database")
             else:
-                query = query.where(MediaItem.id == id)
-            item = session.execute(query).unique().scalar_one()
-        except NoResultFound:
+                logger.debug(f"Item with ID {id} not found in database")
             raise HTTPException(status_code=404, detail="Item not found")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)